### ISSUE TITLE
Swift support on `arm64`

### ISSUE THREAD

**Issue by rvolosatovs** [rvolosatovs/docker-protobuf#89]
Swift is excluded from `arm64` builds currently. See https://github.com/rvolosatovs/docker-protobuf/pull/87 for more details

Possibly related #77

**Comment by strophy** [comment]
Some possible progress here.

Thread on the previous error experienced under Swift 5.6: https://forums.swift.org/t/build-crash-when-building-in-qemu-using-new-swift-5-6-arm64-image/56090

The error has changed under Swift 5.9: https://forums.swift.org/t/swift-runtime-unable-to-suspend-thread-when-compiling-in-qemu/67676

**Comment by strophy** [comment]
Resolved in #351, error still occurs occasionally and has changed to:
```
swift-runtime: unable to suspend thread 915
swift-runtime: unable to suspend thread 1595
swift-runtime: unable to suspend thread 915
swift-runtime: unable to suspend thread 1623

*** Signal 5: Backtracing from 0x55052c0490... failed ***

Trace/breakpoint trap (core dumped)
```
But it is less frequent, and once we have a build in the cache then it doesn't need building again, so let's try and resolve this for now.
